### DISCUSSION TITLE
select options not changing

### DISCUSSION THREAD

**Discussion by burhancoskun10** [nick-keller/react-datasheet-grid#60] [Q&A] [answered]
Hi,
I implemented react select to a column exactly like on the doc. I want options to be dynamic. When I add a measure unit, the unit should be available in the select column. I am adding the unit to the variable where I store the options of the select column but it doesn't show the unit.

**Comment by nick-keller** [comment, marked as answer]
If I understand correctly, you are trying to update the list of options of the select, but the list does not update?
Are you using `<DynamicDataSheetGrid />`? If you need dynamic columns (columns that change after the first render), you need to use `<DynamicDataSheetGrid />`.

More information here: https://react-datasheet-grid.netlify.app/docs/performance/static-vs-dynamic
